src/main/java/uk/gov/hmcts/reform/unspec/model/CaseData.java

```diff
@@ -30,7 +31,9 @@
 @HasServiceDateTheSameAsOrAfterIssueDate(groups = ConfirmServiceDateGroup.class)
 public class CaseData {
 
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
     private final Long ccdCaseReference;
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
     private final CaseState ccdState;
```

Comment:
why are these added to CaseData?

Reply:
These fields are being used already and we need them in our business logic. But we don't want to save these as part of data map in CaseDetails as we don't have any case field for these.